flink/flink-scala-parent/src/main/java/org/apache/zeppelin/flink/sql/AbstractStreamSqlJob.java

```diff
@@ -99,7 +104,7 @@ private static TableSchema removeTimeAttributes(TableSchema schema) {
   protected abstract String getType();
 
   public String run(String st) throws IOException {
-    Table table = stenv.sqlQuery(st);
+    this.table = stenv.sqlQuery(st);
     String tableName = "UnnamedTable_" +
             "_" + SQL_INDEX.getAndIncrement();
     return run(table, tableName);
```

Comment:
`this.table` member variable can not be set normally if `run(Table table, String tableName)` method be called directly.
Is it OK in this scenario?

Reply:
Good point @cuspymd I have set table in this run method as well.